dask/__init__.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import absolute_import, division, print_function
 
+from .utils_test import GetFunctionTestCase
```

Comment:
I'd rather this wasn't in our toplevel namespace. Can we just keep it in `utils_test`?